addons/attributes/initKeybinds.sqf

```diff
@@ -0,0 +1,5 @@
+[ELSTRING(main,DisplayName), QGVAR(attributesDisplay), [LSTRING(AttributesDisplay), LSTRING(AttributesDisplay_Description)], {
+    if (!isNull curatorCamera && {!dialog && {!GETMVAR(RscDisplayCurator_search,false)}}) then {
+        [] call EFUNC(attributes,openForSelection);
```

Comment:
I'd say move all keybind handling code into this file - generally how it is done in other components.

Reply:
Do you mean move fnc_openForSelection into the code for this keybind?

Reply:
Yes.